### PR TITLE
use conventional folder naming

### PR DESCRIPTION
**Pull request by faddat**

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -212,10 +212,10 @@ type MarsApp struct {
 	BankKeeper        bankkeeper.Keeper
 	CapabilityKeeper  *capabilitykeeper.Keeper
 	CrisisKeeper      crisiskeeper.Keeper
-	DistrKeeper       customdistrkeeper.Keeper // replaces the vanilla distr keeper with our custom one
+	DistrKeeper       customdistrkeeper.Keeper // replaces the vanilla distr keeper with our x one
```
Comment (larry0x) [inline app/app.go:215]: jacob, i guess you used search+replacement for this PR, but this doesn't always make sense, for example here

File: app/wasm/msg_plugin.go [hunk 2]
```diff
@@ -35,12 +35,12 @@ var _ wasmkeeper.Messenger = (*CustomMessenger)(nil)
...
 	ctx sdk.Context, contractAddr sdk.AccAddress, contractIBCPortID string, msg wasmvmtypes.CosmosMsg,
 ) ([]sdk.Event, [][]byte, error) {
-	// if the msg is a custom msg, parse it into `MarsMsg` then dispatch to the appropriate Mars module
+	// if the msg is a x msg, parse it into `MarsMsg` then dispatch to the appropriate Mars module
 	// otherwise, simply dispatch it to the wrapped messenger
 	if msg.Custom != nil {
 		var marsMsg MarsMsg
 		if err := json.Unmarshal(msg.Custom, &marsMsg); err != nil {
-			return nil, nil, sdkerrors.Wrapf(err, "invalid custom msg: %s", msg.Custom)
+			return nil, nil, sdkerrors.Wrapf(err, "invalid x msg: %s", msg.Custom)
```
Comment (larry0x) [inline app/wasm/msg_plugin.go:43]: or here

**Comment by larry0x** [comment]
hi jacob, i agree it's better to put the modules into `x`. however you replaced some instances of the word `custom` with `x` where it doesn't really make sense (see the two comments i made for example)

also btw, i initially decided to put them in `custom` folder because terra did this: https://github.com/terra-money/classic-core/tree/main/custom

**Comment by larry0x** [comment]
ok i implemented this change you suggested in commit https://github.com/mars-protocol/hub/commit/3909e1fbcd9d9bbc5061d88a53fa1090a4d7cc86, so this PR is now obsolete @faddat

**Comment by faddat** [comment]
oh lol :D

**Comment by faddat** [comment]
you know, this pr was actually a flub.

I will put some more info on why that is in the one on 46

**Pull request closed without merging**